README.md

```diff
@@ -179,12 +179,13 @@ Elasticsearch](https://github.com/deviantony/docker-elk/wiki/Elasticsearch-clust
 The data stored in Elasticsearch will be persisted after container reboot but not after container removal.
 
 In order to persist Elasticsearch data even after removing the Elasticsearch container, you'll have to mount a volume on
-your Docker host. Update the `elasticsearch` service declaration to:
+your Docker host. Edit ```docker-compose.yml``` and add the path to persistent storage underneath the existing entry in the `elasticsearch` service declaration:
```

Comment:
@seanfcarroll thanks for the suggestion! Your addition is correct but maybe a bit verbose. How about something more succinct like:

```suggestion
your Docker host. Add the following bind mount to the existing volumes of the `elasticsearch` service declaration:
```

Reply:
Ping @seanfcarroll